telegram-bot/main.go

```diff
@@ -120,6 +124,24 @@ func main() {
 	u := tbot.NewUpdate(0)
 	u.Timeout = 60
 	updates, _ := bot.GetUpdatesChan(u)
+	// Set client options
+	clientOptions := options.Client().ApplyURI("mongodb://localhost:27017")
```

Comment:
This is good as of now. But, in future we will need to change it since probably we need to use a cloud database.

Reply:
Should we host the database on Mongo Atlas or something?

Reply:
Yes, we will have to do that. Maybe on Atlas or mlab